master/getting-started/bare-metal/bare-metal.md

```diff
@@ -231,20 +227,15 @@ EOF
 Once you have such a policy in place, you may want to disable the
 [failsafe rules](#failsafe-rules).
 
-> **NOTE**
->
-> Packets that reach the end of the list of rules fall-through to the next policy (sorted by the order field).
->
-> The selector in the policy, `all()`, will match *all* endpoints,
-> including any workload endpoints. If you have workload endpoints as
-> well as host endpoints then you may wish to use a more restrictive
-> selector. For example, you could label management interfaces with
-> label `endpoint_type = management` and then use selector
-> `endpoint_type == "management"`
->
-> If you are using Calico for networking workloads, you should add
-> inbound and outbound rules to allow BGP:  add an ingress and egress rule
-> to allow TCP traffic to destination port 179.
+<div class="alert alert-info" role="alert"><b>Note</b>: Packets that reach the end of the list of rules fall through to the next policy (sorted by the <samp>order</samp> field). 
+<p></p>
+<p></p>
+The selector in the policy, <samp>all()</samp>, will match <b>all</b> endpoints, including any workload endpoints. If you have workload endpoints as well as host endpoints then you may wish to use a more restrictive selector. For example, you could label management interfaces with label <samp>endpoint_type = management</samp> and then use selector <samp>endpoint_type == "management"</samp>
+<p></p>
```

Comment:
I'm afraid I'm really not keen on using `<samp>` here instead of simple backquotes.  I think it would be a step backwards if we had to write more and more HTML in our docs.

Reply:
We cannot use the backticks in the `<div>` elements.